examples/18_check_turbine.py

```diff
@@ -39,6 +39,7 @@
 
 # Get a list of available turbine models
 turbines = os.listdir('../floris/turbine_library')
+turbines = [t for t in turbines if 'yaml' in t]
```

Comment:
@paulf81 It's probably more complete to also support `.yml`.

Reply:
what if I did just .y?  in my case I was getting errors coming from a "DS_STORE" file mac seems to just throw everywhere

Reply:
Ah I just noticed that was in the example and in the [simulation library](https://github.com/NREL/floris/blob/main/floris/simulation/farm.py#L84). That explains why it wasn't failing before. For the example, I think what you have is great.